node/network/statement-distribution/src/metrics.rs

```diff
@@ -18,7 +18,7 @@ use polkadot_node_subsystem_util::metrics::{self, prometheus};
 
 /// Buckets more suitable for checking the typical latency values
 const HISTOGRAM_LATENCY_BUCKETS: &[f64] =
-	&[0.01, 0.025, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.75, 0.9, 1.0, 1.2, 1.5, 1.75];
+	&[0.000025, 0.00005, 0.000075, 0.0001, 0.0003125, 0.000625, 0.00125, 0.0025, 0.005, 0.01, 0.025, 0.05, 0.1];
```

Comment:
Locally these buckets don't make a big difference, but I am interested in seeing if they do in a more realistic scenario.

Reply:
For some realistic values I would look at some historical data from Versi. Even if we don't have these buckets, the sum and count of samples allow us to compute some average value which could be used to estimated some good buckets.

Reply:
Good idea!